.styluaignore

```diff
@@ -0,0 +1,4 @@
+doc/
+plugin/
+.editorconfig
+*.md
```

Comment:
Is this file really needed? Will the GitHub action try to format these files?

Reply:
Is for prevent stylua format other folder and this close and show a exit code but i try test without this file